tarynmartin/personal-portfolio
# Feature/landing page

#### What’s this PR do?
- Change layout of main component and adjust associated styling

#### Where should the reviewer start?
- Main.js to see new layout
- App.js to see changes to router
- About to see how that component will now be rendered/styled

#### How should this be manually tested?
- in terminal, run npm start
- in browser, click to main page and then click on the about button to see how it will look
